src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    public Lottos(List<List<Integer>> numbers) {
+        this.lottos = numbers.stream()
+                .map(Lotto::new)
+                .collect(Collectors.toList());
```

Comment:
JDK 17부터는 `toList()` 메서드 사용이 가능합니다!
```suggestion
                .toList();
```